4-fleetscope/modules/env_baseline/acm.tf

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_sourcerepo_repository" "acm_repo" {
+  project = var.fleet_project_id
+  name    = "eab-acm"
+}
+
+resource "google_service_account" "root-reconciler" {
```

Comment:
nit: here and throughout, can we standardize on underscores for resource and module names

Reply:
done